src/main.js

```diff
@@ -9,5 +9,70 @@ var searchButton = document.querySelector(".search-button");
 var starButton = document.querySelector(".like");
 var deleteButton = document.querySelector(".delete");
 var commentButton = document.querySelector(".comment-button");
-//idea list array
+var ideaGrid = document.querySelector(".idea-grid")
+
 var savedIdeas = [];
+
+// EVENT LISTENERS
+saveButton.addEventListener('click', saveIdea);
+bodyInput.addEventListener('keyup', enableSaveButton);
+
+// FUNCTIONS
+function saveIdea() {
+  if (titleInput.value && bodyInput.value) {
+    var idea = new Idea(titleInput.value, bodyInput.value);
+    savedIdeas.push(idea);
+  }
+  titleInput.value = "";
+  bodyInput.value = "";
+  saveButton.disabled = true;
+  showIdeas();
+  console.log('hot damn')
```

Comment:
I think this is a great function, but we got to remember to delete this!

Reply:
HOT DAMN!